README.md

```diff
@@ -914,6 +915,4 @@ There are 56 apps that you can install on your cluster.
 | [waypoint](https://github.com/hashicorp/waypoint)                            | Easy application deployment for Kubernetes and Amazon ECS                                                                                                       |
 | [yq](https://github.com/mikefarah/yq)                                        | Portable command-line YAML processor.                                                                                                                           |
 | [yt-dlp](https://github.com/yt-dlp/yt-dlp)                                   | Fork of youtube-dl with additional features and fixes                                                                                                           |
-There are 154 tools, use `arkade get NAME` to download one.
-
-> Note to contributors, run `arkade get --format markdown` to generate this list
+There are 155 tools, use `arkade get NAME` to download one.
```

Comment:
Did we intend to drop the note to contributors?

Reply:
No it was a miss.